### ISSUE TITLE
Do not escape underscores

### ISSUE THREAD

**Issue by opoudjis** [metanorma/stepmod-utils#37]
Currently annotated Express files are generating output like:

`The assignment of an identifier corresponds to the *id\_attribute* entity data type.`

This is incorrect Asciidoctor: it actually renders as *id\\_attribute*. The Asciidoctor rules around markup are documented in  https://docs.asciidoctor.org/asciidoc/latest/text/#unconstrained and https://docs.asciidoctor.org/asciidoc/latest/text/troubleshoot-unconstrained-formatting/#use-unconstrained

In short:

* Inside of a word (delimited by space or punctuation), markup punctuation (* _ ``) only needs to be escaped if (a) it is double, and (b) the same pair occurs elsewhere in a string. So `a__b__c` is interpreted as italics markup in Asciidoc, and needs to be escaped as  `a\\__b__c`. But *none* of the following need backslashes: `a_bc`, `a_b_c`, `a__b_c`, `a__bc`.
* The same applies for any word-internal instances of * or `

**Comment by opoudjis** [comment]
It is possible that this is a bug in ReverseAsciidoctor, where this code originates from. If so, this bug fix needs to be applied back to ReverseAsciidoctor.
